src/pages/home.tsx

```diff
@@ -41,7 +41,10 @@ export const HomePage: React.FunctionComponent = () => {
           <p className="text-red-600 font-medium">Verifiable Credential is invalid</p>
         </div>
       ) : null}
-      {documentStatus === Status.RESOLVED ? <DocumentRenderer rawDocument={rawDocument as WrappedDocument} /> : null}
+      {/* {documentStatus === Status.RESOLVED ? <DocumentRenderer rawDocument={rawDocument as WrappedDocument} /> : null} */}
+      {documentStatus !== Status.PENDING && documentStatus !== Status.IDLE ? (
```

Comment:
why changing this ?

Reply:
only for testing to show tabs using opencerts file with tabs available

Reply:
i have reverted the change